care/facility/events/handler.py

```diff
@@ -58,6 +58,15 @@ def create_consultation_event_entry(
     ).values_list("id", "fields")
     for group_id, group_fields in groups:
         if set(group_fields) & fields_to_store:
+            value = {}
+            for field in group_fields:
+                try:
+                    value[field] = data[field]
+                except KeyError:
+                    value[field] = getattr(object_instance, field, None)
+            # if all values in the group are Falsy, skip creating the event for this group
+            if all(not v for v in value.values()):
```

Comment:
Should we consider making the check against None instead?
What if any of the vital is 0 (falsy)?

Reply:
yeah makes sense, will update it